src/forms/CurrentExpenses.js

```diff
@@ -286,12 +297,16 @@ const ExpensesFormContent = function ({ current, time, setClientProperty }) {
             { ...sharedProps }
             generic={ 'childOtherCare' }> Other care 
           </CashFlowRow>
-          <CashFlowRowAfterConfirm
-            { ...sharedProps }
-            generic={ 'earnedBecauseOfChildCare' }
-            confirmLabel={ 'Does childcare allow you to make additional income?' }>
-            <span style={{ textDecoration: 'underline' }}>Income</span> made possible by child care expenses
-          </CashFlowRowAfterConfirm>
+
+          <EarnedFrom
+            heading={ 'Effect of Childcare' }
+            cashflowProps={{
+              ...sharedProps,
+              generic:      'earnedBecauseOfChildCare',
+              confirmLabel: 'If you didn\'t have that childcare, would it affect your current job?',
```

Comment:
I'd get rid of the word "that," since it sounds like you're assuming they do get child care currently. Also maybe replace the word "job" with "income" to make it a little less vague.

Reply:
Hmm, what we need here is to only show that option if the person has child care. First, then the 'that' would make sense. Second, it really doesn't make sense to ask that if there's no child care expense. It also might be a good way to get them to notice the question about income earned from child care. I was hoping to avoid that annoyance, but maybe we'll need to add it in.

The reason it's 'that' child care is because people also get their child care subsidized, but we're not asking about the income they make from subsidized child care, just the child care they listed above.

We used 'job' instead of 'income' because it's a word with which more people are familiar. I have to admit it doesn't feel vague to me. The way I think about it, having to work fewer hours or not being able to keep your job seem to be well covered by it, but maybe I've been playing with the wording too long to see it objectively. We played around with the wording for a while, but there are probably better options. Can you think of any that stick to plainer language?

Reply:
The reason it feels vague to me is someone might think "well my job would be the same, I'd just work fewer hours" and answer no. I honestly think "income" is a pretty simple word for someone who has a job. Would "wages" be better? Or "pay"?

Reply:
I don't like the word 'affect' either, btw. Still trying to figure out that wording.

Reply:
Great words! I just had a thought, though. How about, 'would it change how much money you bringing home'?

Reply:
Dang, this is an 'outdated' hidden thread now. Damn you, github!